### PR TITLE
feat(lerna): added independentMode

### PR DESCRIPTION
**Pull request by Dazza65**
Allows the user to specify Lerna [independent version mode](https://lerna.js.org/docs/features/version-and-publish#independent-mode) to independently version sub-projects

### CODE REVIEW

File: .projenrc.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import {javascript, cdk} from 'projen'
 
-const repository = 'https://github.com/AminFazlMondo/Lerna-Projen.git'
+const repository = 'https://github.com/Dazza65/Lerna-Projen.git'
```
Comment (AminFazlMondo) [inline .projenrc.ts:3]: Could you please revert the changes regarding the use of your user name?

**Review by AminFazlMondo — CHANGES_REQUESTED** [review]

**Comment by AminFazlMondo** [comment]
Thanks @Dazza65  for contributing into this repository.

File: .projenrc.ts (hunk 1 above)
Comment (Dazza65) [reply .projenrc.ts]: Done.

Comment (AminFazlMondo) [reply .projenrc.ts]: There are more references to your user name in some workflow files and package.json

**Comment by Dazza65** [comment]
Apologies for the other changes, which were made when I built & tested my changes.  I was intending on adding the fork to my own npm repo as I wasn't sure if you'd accept the PR but you've been super responsive 🥇

**Comment by Dazza65** [comment]
I'd checked in the API.md which was causing the build to fail so have also reverted that.

File: .projenrc.ts (hunk 1 above)
Comment (Dazza65) [reply .projenrc.ts]: Should just be the 3 files changed now: src/index.ts, src/types.ts and test/index.test.ts

**Comment by AminFazlMondo** [comment]
Could you please apply changes from patch artifact https://github.com/AminFazlMondo/Lerna-Projen/suites/7812265867/artifacts/329911261?

**Review by AminFazlMondo — APPROVED** [review]

**Comment by Dazza65** [comment]
Thanks for your patience,  it probably consumed more time in your review than it would have to commit the changes yourself - lol
